engine/src/main/scala/cromwell/services/MetadataServiceActor.scala

```diff
@@ -0,0 +1,89 @@
+package cromwell.services
+
+import akka.actor.Actor
+import cromwell.engine.workflow.WorkflowManagerActor.WorkflowMetadata
+import cromwell.engine.{ExecutionEventEntry, WorkflowId}
+import cromwell.engine.backend.JobKey
+import cromwell.services.MetadataServiceActor._
+import cromwell.webservice.WorkflowMetadataResponse
+import org.joda.time.DateTime
+import wdl4s.values.WdlValue
+
+object MetadataServiceActor {
+
+  sealed trait MetadataEvent[V] {
```

Comment:
I'd assumed this service was just taking arbitrary metadata values from any requester and the ability to pull them back out, leaving responsibility for understanding the semantics of anything in there to someone else. Is that not how it's been drawn up?